context-free.md

```diff
@@ -2,21 +2,27 @@
 
 ##  Who is this programming language for?
 
+This programming language appears to be for anyone interested in vector-based digital art that does not want to go through the hassle of learning a more complete graphics editor, like Photoshop. Although the language itself is not extremely complicated, it seems geared toward someone who has some programming experience.
```

Comment:
I would consider how this language differs than those that you mention. While it accomplishes vector-based digital art, it isn't necessarily for people who want to draw every type of image, right? Its really easy to do recursion and automation, so think of how those features of the language alter the intended user.